docs/usage/ipv6.md

```diff
@@ -189,14 +164,9 @@ spec:
 ---
 
 ## Summary
```

Comment:
I see no need for this section as well. If we want to keep parts of the content, let's move it to the other sections.